public/js/monitor.js

```diff
@@ -36,19 +37,100 @@ function isValidEmail(val) {
 }
 
 
-function doOauth(el) {
+function doOauth(el, {emailWatch = false} = {}) {
+  // Growth Experiment: To sidestep the breach scans form, we have to check if
+  // there is an email address entered into #scan-user-email form.
+  // If so, we set it similiar to what would happen on form submit.
+  //
+  // Options was added to limit how the watched email input field is injected
+  // in this function. It only moves it if  options.emailWatch is set to TRUE;
+
   let url = new URL("/oauth/init", document.body.dataset.serverUrl);
   url = getFxaUtms(url);
+
   ["flowId", "flowBeginTime", "entrypoint"].forEach(key => {
```

Comment:
We'll want to add `entrypoint_experiment` and `entrypoint_variation` to this, otherwise those values won't make it to the OAuth flow. Since not every entrypoint element will have `entrypoint_experiment` and `entrypoint_variation` attributes we'll need to check before appending to the url though.